includes/class-newspack-newsletters-ads.php

```diff
@@ -87,7 +87,7 @@ public static function rest_api_init() {
 	 * @return bool|WP_Error
 	 */
 	public static function permission_callback( $request ) {
-		return current_user_can( 'edit_posts' );
+		return current_user_can( 'edit_' . self::CPT . 's' );
```

Comment:
We need to use that `$post_type_object->caps->edit_posts` method here instead of hard coding this. This appears in many places in the PR

It's much more readable and avoid human errors when adding new checks in the future

Reply:
Good idea, done in f20f06e2b8aa2b4db31af1ff3624f5945fd60e6d 👌